var/spack/repos/builtin/packages/mpich/package.py

```diff
@@ -27,7 +27,7 @@ class Mpich(AutotoolsPackage, CudaPackage, ROCmPackage):
 
     keep_werror = "specific"
 
-    license("Unlicense")
+    license("mpich2")
```

Comment:
Interesting.  I haven't seen any licenses like this before.  I noticed the download page (https://www.mpich.org/downloads/) says it is a `BSD-like` license so I was initially expecting something maybe along the lines `BSD-3-Clause AND MIT` or `MIT AND BSD-3-Clause-Modificaiton`.

 But I found a link to it on `spdx.org` for MIT licensing and noticed the `mpich2` variant (https://fedoraproject.org/wiki/Licensing:MIT#mpich2_variant) ) .  So I'm *wondering* if something like `MIT-mpich2` is appropriate?

Reply:
Thanks for reviewing! I only did this because I came across the `license("Unlicense")` in the code and got curious enough to on an expedition 😄.

I was surprised to see `mpich2` on SPDX https://spdx.org/licenses/mpich2.html, which did come via Fedora from the looks of it. I should update our download page. I think adding some clue that the `mpich2` license is similar to MIT sounds reasonable.

Reply:
Ping me when you're ready for this PR to be merged.